config/deploy.rb

```diff
@@ -38,4 +38,35 @@
       end
     end
   end
+
+  desc 'Confirm if you really want to execute the task'
+  task :confirmation do
+    puts <<-WARN
+
+    ===============================================================================
+
+      WARNING: You're about to run tasks on #{ENVS_FOR_CONFIRMATION_STEP.join('/')} server(s)
+      Please confirm that all your intentions are kind and friendly.
+
+      Check if:
+
+      * You are deploying the correct branch
```

Comment:
Let's make this more explicit, such as:
```suggestion
      This will deploy `#{BRANCH}` to #{ENV})
      
      Check if:
      * You are deploying the correct branch
      * You are deploying to the correct environment
```